src/components/ContentWrapper/ContentWrapper.module.scss

```diff
@@ -0,0 +1,122 @@
+@import "styles/Variables";
+
+.content {
```

Comment:
Now that we have styles for ContentWrapper we might consider removing the `Gutenburg.scss` styles and the related dependancies.

Reply:
@theodesp Great point. The styles in this PR really only handle non-block styles, i.e. classic editor styles. Some of these classic editor styles do cascade into the block styles, but the Gutenberg.scss styles are still necessary at this time.